ClimateEmergency_Engine/Compute/CalculateAssemblies/ReinforcementVolume.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Base;
+using BH.oM.Reflection.Attributes;
+using BH.oM.Structure.MaterialFragments;
+
+namespace BH.Engine.ClimateEmergency
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Returns reinforcement volume.")]
+        [Input("concreteVolume", "Volume of concrete building element m^3.")]
+        [Input("percentageReinforcement", "Percentage reinforcement within concrete building element m^3.")]
+        [Output("ReinforcementVolume", "Percentage reinforcement within concrete building element m^3.")]
+        public static double ReinforcementVolume(double concreteVolume, double percentageReinforcement)
+        {
+            return concreteVolume * (percentageReinforcement * .01);
```

Comment:
Multiplication should be `0.01` rather than just `.01`. I'm assuming this is to convert the percentage of reinforcement from a percentage of 0-100 into 0-1 for the multiplication? If so, this should probably be stated in the documentation input for `percentageReinforcement` to make it clear to users that this function is expecting a number between 0 and 100.